event/processor.go

```diff
@@ -115,7 +116,11 @@ func (p *eventProcessor) run(ctx context.Context, tail bool) error {
 		return err
 	}
 
-	defer list.Destroy(context.Background())
+	defer func() {
+		if err := list.Destroy(context.Background()); err != nil {
+			log.Fatal(err)
```

Comment:
We don't want to `log.Fatal` as that will terminate the program.  Using `log.Print` is fine.  May want to add some context to the message `"destroy event view: %s"`